app/javascript/application.js

```diff
@@ -37,4 +37,12 @@ document.addEventListener("DOMContentLoaded", function() {
       }
     }
   });
+
+  document.addEventListener("click", function(event) {
+    if (event.target.closest('.like-form')) {
+      var likeForm = event.target.closest('.like-form');
+      var submitButton = likeForm.querySelector('input[type="submit"]');
+      submitButton.click();
+    }
+  });
```

Comment:
Is this needed?

Reply:
I included this because the submit button is hidden. So this way by clicking the heart icon the submit button is going to be clicked (otherwise clicking the icon would have no effect, the like would not be created)

Reply:
You can make the heart icon work as a submit button itself